src/main/java/uk/gov/hmcts/ccd/domain/service/startevent/DefaultStartEventOperation.java

```diff
@@ -99,6 +106,9 @@ public StartEventResult triggerStartForCase(final String caseReference,
 
```

Comment:
mmm perhaps line 99 
`final CaseDetails caseDetails = getCaseDetails(caseReference);`
should already return the case with the default values? We should change `getCaseDetails` to use the Service rather than the Repository directly, and have the CaseService dealing with defaulting

Reply:
we can ignore this